File: new/atoms/sup-name-keys.test.mjs
import * as m from './sup-name-keys.mjs';
const SNAP = {"SUP_NAME_KEYS":"[\"שם\",\"תורם\"]"};
let f = 0;
for (const [k, s] of Object.entries(SNAP)) if (JSON.stringify(m[k]) !== s) { console.error('✗ ' + k + ' סטה מהצילום'); f = 1; }
if (f) process.exit(1); console.log('✓ sup-name-keys: צילום-ערך תואם — ירוק');

File: new/dart-maor/sup-name-keys_test.dart
// בדיקת-חוזה (רתמת-זהב) · sup-name-keys — צילום-ערך jsonEncode≡JSON.stringify.
// חוללה מכנית (gen-data-dart). הרצה: dart run --enable-asserts ⇒ OK
import 'dart:convert';
import 'sup-name-keys.dart';

void main() {
  // ‏1) צילום-ערך: jsonEncode ≡ JSON.stringify של המקור (זהה-ביט).
  if (jsonEncode(supNameKeys) != '["שם","תורם"]') {
    throw StateError('FAIL supNameKeys: צילום-הערך סטה');
  }
  print('OK sup-name-keys: 1 data-snapshot(s) passed');
}
